python/graphstorm/eval/evaluator.py

```diff
@@ -1020,6 +1020,7 @@ def compute_score(
         # compute ranking value for each metric
         metrics: Dict[str, th.Tensor] = {}
         for metric in self.metric_list:
+            # NOTE: If other metrics needs candidate list sizes, add them here.
```

Comment:
```suggestion
            # NOTE: If other metrics needs candidate list sizes, add them here.
            # NOTE: aoid adding the size twice to avoid possible errors.
```